### ISSUE TITLE
Import of translated order failed.

### ISSUE THREAD

**Issue by joshuakeck** [AcclaroInc/craft-translations#421]
Hi guys,
we currently have problems to import a translated order via manual import.
We upload the zip file at the order, and receive the following error after clicking on "Merge and apply draft":
"Error: foreach() argument must be of type array|object, string given".

We try to translate a german site into english.

Any idea?

These are the specs of the project:
Craft version: 4.3.8.2
PHP version: 8.1.12
MultiSite: true

**Plugins:**
Admin Bar = 3.2.0
Cache Warmer = 2.1.2
Commerce Currency Prices = 3.0.0-beta.4
Control Panel CSS = 2.6.0
Craft Commerce = 4.2.5.1
Dumper = 3.0.1
Element API = 3.0.1.1
Elements Panel = 2.0.0
GeoMate = 2.1.0
Image Hotspots = 2.0.0-beta.1
Mollie for Craft Commerce = 4.1.0
Neo = 3.6.3
Redactor = 3.0.3
Relax = 2.0.1
SEOMatic = 3.0.20
Servd Assets and Helpers = 3.2.7
Site Copy X = 1.0.4
Verbb Super Table = 3.0.7
Acclaro Translations = v3.2.1
Mutation Digitale Translations admin = 3.0.1
NYStudio107 Webperf = 4.0.0
Verbb Wishlist = 2.0.4

**Comment by shnsumit** [comment]
@joshuakeck Hi,

This problem could be caused by incorrect data in the files you're attempting to import. It might be not able to fetch expected data. 

To debug this further can you please share the logs and files the you are trying to import?

Thanks

**Comment by joshuakeck** [comment]
@shnsumit, thanks for the quick response!
Attached, you can find the files I am trying to upload. 
[pseudo-translated_post.zip](https://github.com/AcclaroInc/craft-translations/files/10795884/pseudo-translated_post.zip)

The log:
```
2023-02-21@18:11:05
php-staging
any errors.'\n 'cp-notification-error' => [\n 0 => 'Error: foreach() argument must be of type array|object, string given'\n 1 => [\n 'icon' => 'alert'\n 'iconLabel' => 'Error'\n ]\n ]\n]"}
```

Thanks!

**Comment by shnsumit** [comment]
@joshuakeck  Hi,

Files looks fine and was not able to find any issue in that. 
Above provided log screenshot was not very helpful. Is it possible for you to send the complete log file to sumit@semantic.dev ?
Make sure to enable the debugging from Translation plugin settings (`Translation -> Settings -> Configuration Options -> Advacned`) . If it was not enabled before then try to reproduce the issue after enabling debugging so that it will log proper details. Also, You can download the logs from `Translation -> Settings -> Send Logs` 

Or It would be nice if we can get access of staging server to look more into this issue.

Thanks!

**Comment by shnsumit** [comment]
@joshuakeck  Resolved in `V3.2.2`
